contracts/interfaces/IGitConsensus.sol

```diff
@@ -126,7 +126,7 @@ interface IGitConsensusTypes {
 ////        unit testing purposes.
 interface IGitConsensus is IGitConsensusErrors, IGitConsensusEvents, IGitConsensusTypes {
     /// @notice Notarizes a commit on-chain, building the hash trustlessly from the commit data.
-    ///     Stores the commit such that `commitExists()` returns true for this commit hash, and
+    ///     Stores the commit such that `hashExists()` returns true for this commit hash, and
     ///     `commitAddr()` returns the owner address. Emits a `IGitConsensusEvents.CommitAdded`
```

Comment:
`commitAddr()` returns the owner address. 

should be :

`hashAddr()` returns the owner address.